main.css

```diff
@@ -0,0 +1,358 @@
+    background-attachment: fixed;
+    background-size: cover;
+    margin: 0;
+    padding: 0;
+}
+
+#homePage{
+    display: grid;
+    grid-template-columns: 20vw 20vw 20vw 20vw 20vw;
+    grid-template-rows: auto;
+    grid-template-areas: 
+    "header header header header header"
+    "restlist map map map map "
+    "footer footer footer footer footer"
+}
+
+li{
+    border-radius: 5%;
+    max-width: 20vw;
+    padding: 3px;
+    text-align: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 20px;
+    height: auto;
+    width: auto;
+    border: 1px black solid;
+    margin: 10px;
+    background-color:white;
+    box-shadow: 2px 2px 2px black;
+}
+li:hover{
+    box-shadow: 4px 4px 4px black;
+}
+
+@media (max-width: 600px){
+    #yelp{
+        font-size: 50px !important;
+    }
+    #bars{
+        font-size: 15px !important;
+    }
+    li{
+        font-size:15px;
+    }
+}
+@media (max-width: 420px){
+    #yelp{
+        font-size: 30px !important;
+    }
+    li{
+        max-height:10vh;
+        font-size:10px;
+    }
+    #name{
+        font-size: 15px;
+    }
+}
+#title{
+    font-weight: 800;
+    box-shadow: 3px 3px 3px black;  
+    margin-bottom: 10px;
+    background-color: #dadada;
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='100' height='100' viewBox='0 0 200 200'%3E%3Cdefs%3E%3ClinearGradient id='a' gradientUnits='userSpaceOnUse' x1='100' y1='33' x2='100' y2='-3'%3E%3Cstop offset='0' stop-color='%23000' stop-opacity='0'/%3E%3Cstop offset='1' stop-color='%23000' stop-opacity='1'/%3E%3C/linearGradient%3E%3ClinearGradient id='b' gradientUnits='userSpaceOnUse' x1='100' y1='135' x2='100' y2='97'%3E%3Cstop offset='0' stop-color='%23000' stop-opacity='0'/%3E%3Cstop offset='1' stop-color='%23000' stop-opacity='1'/%3E%3C/linearGradient%3E%3C/defs%3E%3Cg fill='%23b9b9b9' fill-opacity='0.6'%3E%3Crect x='100' width='100' height='100'/%3E%3Crect y='100' width='100' height='100'/%3E%3C/g%3E%3Cg fill-opacity='0.5'%3E%3Cpolygon fill='url(%23a)' points='100 30 0 0 200 0'/%3E%3Cpolygon fill='url(%23b)' points='100 100 0 130 0 100 200 100 200 130'/%3E%3C/g%3E%3C/svg%3E");
+    font-family: Arial, Helvetica, sans-serif;
+    text-align: center;
+    grid-area: header;
+}
+#yelp{
+    font-weight: 800;
+    font-size: 70px;
+    color: rgb(57, 109, 50);
+    text-shadow: 2px 2px 2px black;
+}
+#bars{
+    font-weight: 800;
+    font-size: 25px;
+}
+#bars:hover{
+    text-decoration: underline;
+}
+#yelp:hover{
+    text-decoration: underline;
+    text-decoration-color: green;
+}
+#rests-list{
+    align-items: center;
+    justify-items: center;
+    grid-area: restlist;
+    max-height: 75vh;
+    overflow: scroll;
+}
+
+#map{
+    margin: 15px;
+    margin-right: 25px;
+    grid-area: map;
+    border-radius: 5%;
+}
+/*-------------------------------------------------------------------------------------*/
```

Comment:
I see you're blocking out sections of your CSS, but without words I don't know what these sections mean.